test-website/chinese-encoding.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="gb2312">
+    <title>Test website</title>
+    <link rel="apple-touch-icon" sizes="180x180" href="./icons/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="./icons/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="./icons/favicon-16x16.png">
+    <link rel="manifest" href="./icons/site.webmanifest">
+    <link rel="shortcut icon" href="./icons/favicon.ico">
+    <link rel="stylesheet" href="./css-broken-file.css">
+</head>
+
+<body>
+
+    <h2>Chinese encoding</h2>
+
+    <p>
+        <img src="http://www.mediaelementjs.com/images/big_buck_bunny.jpg">
+    </p>
+
+    <p>This file has been encoded with Simplified Chinese gb2312 encoding and simplified chinese characters: ����.</p>
```

Comment:
And firefox decode it as `GBK`...

Reply:
This is specific to one browser, is it really worth it to mention this?